app/services/current-user.js

```diff
@@ -59,8 +62,22 @@ export default Ember.Service.extend({
      */
     user: Ember.computed('currentUserId', function() {
         const ObjectPromiseProxy = Ember.ObjectProxy.extend(Ember.PromiseProxyMixin);
+        this.get('_setWaflle').perform();
         return ObjectPromiseProxy.create({
             promise: this.load().catch(() => null),
         });
     }),
+
+    _setWaflle: task(function* () {
+        const url = `${config.OSF.apiUrl}/v2/_waffle/`;
+        const { data } = yield $.ajax(url, 'GET');
```

Comment:
What do you mean by not a logged-in request?

Reply:
Like including the cookie, so the API knows what user to get the waffle for. But I just realized that's only a concern for cross-origin ajax; it'll do the right thing by default for same-origin. In case we ever use this code in other apps, though, we should probably use `authenticatedAjax` from `ember-osf-web/utils/ajax-helpers` for all our ajax needs.